doc/changelog.md

```diff
@@ -2,6 +2,148 @@
 
 <h3>New features</h3>
 
+* Add support for accelerating classical processing via JAX with `catalyst.accelerate`.
+  [(#805)](https://github.com/PennyLaneAI/catalyst/pull/805)
+
+  Classical code that can be just-in-time compiled with JAX can now be seamlessly just
+  in time compiled with `catalyst.accelerate` and included within QJIT-compiled functions.
+  `catalyst.accelerate` can be used as a
+  decorator without specifying a device:
+
+  ```python
+  @accelerate(dev=jax.devices("gpu")[0])
+  def classical_fn(x):
+      return jnp.sin(x) ** 2
+
+  @qjit
+  def hybrid_fn(x):
+      y = classical_fn(jnp.sqrt(x)) # will be executed on a GPU
+      return jnp.cos(y)
+  ```
+
+  Available devices can be retrieved via
+  `jax.devices()`. If not provided, the default value of
+  `jax.devices()[0]` as determined by JAX will be used.
+
+* Catalyst callback functions, such as `pure_callback`, `debug.callback`, and `debug.print`, now
+  all support autodifferentiation.
+  [(#706)](https://github.com/PennyLaneAI/catalyst/pull/706)
+  [(#782)](https://github.com/PennyLaneAI/catalyst/pull/782)
+  [(#822)](https://github.com/PennyLaneAI/catalyst/pull/822)
+  [(#834)](https://github.com/PennyLaneAI/catalyst/pull/834)
+
+  - When using callbacks that do not return any values, such as `catalyst.debug.callback` and
+    `catalyst.debug.print`, these functions are marked as 'inactive' and do not contribute to or
+    affect the gradient of the function:
+
+    ```python
+    import logging
+  
+    log = logging.getLogger(__name__)
+    log.setLevel(logging.INFO)
+  
+    @qml.qjit
+    @catalyst.grad
+    def f(x):
+        y = jnp.cos(x)
+        catalyst.debug.print("Debug print: y = {0:.4f}", y)
+        catalyst.debug.callback(lambda _: log.info("Value of y = %s", _))(y)
+        return y ** 2
+    ```
+  
+    ```pycon
+    >>> f(0.54)
+    INFO:__main__:Value of y = 0.8577086813638242
+    Debug print: y = 0.8577
+    array(-0.88195781)
+    ```
+
+  - Callbacks that *do* return values and may affect the qjit-compiled functions
+    computation, such as `pure_callback`, must have custom gradients manually
```

Comment:
```suggestion
    computation, such as `pure_callback`, may have custom gradients manually
```

If the user uses a `pure_callback` somewhere that is not reachable from a `grad` op, then there is no need to register custom gradients. Not sure if you want this correction. Just thought it is an important distinction.